src/clusterfuzz/_internal/tests/appengine/libs/issue_filer_test.py

```diff
@@ -453,7 +455,8 @@ def my_save(*args, **kwargs):
     setattr(my_save, 'raise_exception', True)
     issue_tracker = monorail.IssueTracker(IssueTrackerManager('chromium'))
     self.mock.get.return_value = CHROMIUM_POLICY
-    issue_filer.file_issue(self.testcase1, issue_tracker)
+    with self.assertRaises(Exception):
+      issue_filer.file_issue(self.testcase1, issue_tracker)
```

Comment:
@Navidem I missed this in the last review, but the correct behaviour here should've been to raise the exception if we fail on the first save() without a fallback.